common/conf/conf.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <common/conf.hpp>
+
+#include <string>
+#include <cstdlib>
+
+namespace mender {
+namespace common {
+namespace conf {
+
+using namespace std;
+
+string GetEnv(const string &var_name, const string &default_value) {
+	const char *value = getenv(var_name.c_str());
+	if (value == nullptr) {
```

Comment:
How badly did you wanna use the ternary?

Reply:
It didn't even cross my mind, actually.